plugins/basic/python/tk_maya_basic/plugin_engine.py

```diff
@@ -83,12 +83,14 @@ def bootstrap(sg_user, progress_callback, completed_callback, failed_callback):
 
     logger.debug("Will launch the engine with entity: %s" % entity)
 
-    # Remove the custom logging handler now that the engine will take over logging.
-    sgtk.LogManager().root_logger.removeHandler(plugin_logging_handler)
-
     # Install the bootstrap progress reporting callback.
     toolkit_mgr.progress_callback = progress_callback
 
+    # install a callback that turns off logging just before the
+    # engine starts up its own logging
+    callback = lambda ctx, log_handler=plugin_logging_handler: _pre_engine_start_callback(ctx, log_handler)
```

Comment:
@jfboismenu is this possible? the `pre_engine_start_callback` runs before the engine exists - how could we do a 'handover' without having the engine instance? Sounds interesting, but we need to do it in a way so that the engine doesn't have a dependency on the plugin! Sorry if i am missing something obvious :)